engine/src/dev/java/org/terasology/benchmark/entitySystem/EntityCreateBenchmark.java

```diff
@@ -27,9 +27,6 @@
 
 import java.util.List;
 
-/**
```

Comment:
While empty javadoc removal is fine for the GCI task as noted in the task each instance only counts for 1/10th to the requirements for the task - since it is trivial to delete a few lines.